src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,43 @@
+package tests;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+
+/*
+A4 – Homework – Refactor your tests using the advantages of TestNG framework:
+JAVA AUTOMATION A4  A4 – Homework – Refactor your tests using the advantages of TestNG framework:
+
+Refactor your tests using the advantages of TestNG framework:
+1. Create Before Method and After methods to close and open browser
+2. Create BaseTest file and move setup and teardown there
+*/
+
+public class BaseTest {
+    public WebDriver driver;
```

Comment:
It should be private or protected variable. Please use encapsulation principle (OOP)